decide/utils.cpp

```diff
@@ -12,7 +14,7 @@ double angle_between_points(Coordinate A, Coordinate B, Coordinate C) {
 
   double a = atan2(B.y - A.y, B.x - A.x);
   double b = atan2(C.y - A.y, C.x - A.x);
-  return b - a;
+  return abs(b - a);
```

Comment:
Alternatively, instead of using `abs()`, this could be solved by adding `pi`. I don't think it makes a difference for this program.

Reply:
Ah, I see. I think `abs` is clearer since we don't care which line represents the x-axis (if we think of a negative angle as extending below the x-axis, that is), but even if we were to change it, shouldn't it be `2*pi`?

Reply:
You are right, it would be adding 2pi as you say, but only if the result of `b - a` is negative. otherwise do nothing. But yes let's leave it as is